packages/@aws-cdk/aws-rds/lib/engine-version.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * A version of an engine -
+ * for either a cluster, or instance.
+ */
+export interface EngineVersion {
+  /**
+   * The full version string of the engine,
+   * for example, "5.6.mysql_aurora.1.22.1".
+   * It can be undefined,
+   * which means RDS should use whatever version it deems appropriate for the given engine type.
```

Comment:
this sounds a little ominous - can a resource be replaced / changed from beneath a customer if they decide to go with the default?

Alternatively it could be required and some string like `LATEST` or similar can be used to enable this mode. It depends on what the chances of getting bitten by the default is as engine types change.